public/js/src/module/form-cache.js

```diff
@@ -102,11 +184,11 @@ function _setUpdateIntervals( survey ) {
     // that open the form right after the XForm update.
     setTimeout( () => {
         _updateCache( survey );
-    }, 3 * 1000 );
+    }, CACHE_UPDATE_INITIAL_DELAY );
```

Comment:
These delays were extracted to constants both for clarity and to aid testing with fake timers.